spec/serializers/site_serializer_spec.rb

```diff
@@ -214,8 +214,8 @@
 
     it "is not included in the serialised object when user is not anonymous" do
       guardian = Guardian.new(user)
-
       serialized = described_class.new(Site.new(guardian), scope: guardian, root: false).as_json
+      expect(serialized).not_to have_key(:anonymous_sidebar_sections)
```

Comment:
This is unrelated to the fix, but found this spec that wasn't doing anything while trying to understand the code. So I decided to fix it.